packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -171,7 +171,7 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
                     webpackImporter: false,
                     implementation: require('node-sass'),
                     sassOptions: {
-                      outputStyle: 'nested',
+                      outputStyle: 'compressed',
```

Comment:
I didn't realize this was being applied to both the dist and dev build, maybe we should flip between `compressed` and `nested` based on `worker.dist`?

Reply:
Pushed with 3fc86d1